apps/server/src/modules/tool/external-tool/mapper/external-tool-datasheet.mapper.spec.ts

```diff
@@ -32,18 +34,57 @@ describe(ExternalToolDatasheetMapper.name, () => {
 				isDeactivated: true,
 				restrictToContexts: [ToolContextType.COURSE, ToolContextType.BOARD_ELEMENT],
 			});
+			const schoolExternalTool: SchoolExternalTool = schoolExternalToolFactory.build({
+				status: { isDeactivated: true },
+			});
 			const datasheet: ExternalToolDatasheetTemplateData = externalToolDatasheetTemplateDataFactory
 				.withOptionalProperties()
 				.withParameters(1, { properties: 'optional, geschützt' })
 				.build({ instance: 'dBildungscloud' });
 
-			return { user, externalTool, datasheet };
+			return { user, externalTool, schoolExternalTool, datasheet };
+		};
+
+		it('should map all parameters correctly', () => {
+			const { user, externalTool, schoolExternalTool, datasheet } = setup();
+
+			const mappedData: ExternalToolDatasheetTemplateData =
+				ExternalToolDatasheetMapper.mapToExternalToolDatasheetTemplateData(
+					externalTool,
+					schoolExternalTool,
+					user.firstName,
+					user.lastName
+				);
+
+			expect(mappedData).toEqual(datasheet);
+		});
+	});
+
+	describe('when tool is deactivated on school level', () => {
+		const setup = () => {
+			const user: UserDO = userDoFactory.build();
+			const externalTool = externalToolFactory.build();
+			const schoolExternalTool: SchoolExternalTool = schoolExternalToolFactory.build({
+				status: { isDeactivated: true },
+			});
+			const datasheet: ExternalToolDatasheetTemplateData = externalToolDatasheetTemplateDataFactory.build({
```

Comment:
Its good practice to name variables which you are using only for expectation something like expectedDatasheet for example